plugin-infra/plugin-metadata-store/src/com/thoughtworks/go/plugin/access/authorization/AuthorizationMetadataStore.java

```diff
@@ -56,4 +56,8 @@ public static AuthorizationMetadataStore instance() {
         }
         return plugins;
     }
+
+    public Set<String> getPluginsThatSupportsWebBasedAuthentication() {
```

Comment:
I thought about it, but since `getPluginsThatSupports` returned a `Set<String>`, I left it as it. But sure this could be changed